src/main/java/bio/terra/app/controller/SnapshotAccessRequestApiController.java

```diff
@@ -105,6 +107,33 @@ public ResponseEntity<SnapshotAccessRequestResponse> approveSnapshotAccessReques
     return ResponseEntity.ok(snapshotBuilderService.approveRequest(id));
   }
 
+  @Override
+  public ResponseEntity<SnapshotAccessRequestMembersResponse> getSnapshotAccessRequestGroupMembers(
+      UUID id) {
+    AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
+    iamService.verifyAuthorization(
+        userRequest, IamResourceType.SNAPSHOT_BUILDER_REQUEST, id.toString(), IamAction.GET);
+    return ResponseEntity.ok(snapshotBuilderService.getGroupMembers(id));
+  }
+
+  @Override
+  public ResponseEntity<SnapshotAccessRequestMembersResponse> addSnapshotAccessRequestGroupMember(
+      String memberEmail, UUID id, PolicyMemberRequest body) {
+    AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
+    iamService.verifyAuthorization(
+        userRequest, IamResourceType.SNAPSHOT_BUILDER_REQUEST, id.toString(), IamAction.APPROVE);
+    return ResponseEntity.ok(snapshotBuilderService.addGroupMember(id, memberEmail));
+  }
+
+  @Override
+  public ResponseEntity<SnapshotAccessRequestMembersResponse>
+      deleteSnapshotAccessRequestGroupMember(UUID id, String memberEmail) {
+    AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
+    iamService.verifyAuthorization(
+        userRequest, IamResourceType.SNAPSHOT_BUILDER_REQUEST, id.toString(), IamAction.APPROVE);
```

Comment:
These look good! I wonder if we need to make a new IAmAction for these actions or if its okay to extend the use of APPROVE for now.